### ISSUE TITLE
Mailcow reporting wrong host name - even after changing in the conf file

### ISSUE THREAD

**Issue by gtmadev** [mailcow/mailcow-dockerized#3157]
**Prior to placing the issue, please check following:** *(fill out each checkbox with a `X` once done)*
- [x ] I understand that not following below instructions might result in immediate closing and deletion of my issue.
- [ x] I have understood that answers are voluntary and community-driven, and not commercial support.
- [ x] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

---

**Mailcow reporting incorrect (old) server host**

After migrating to a new server, the host name was changed in mailcow.conf. I have also forced an update, restarted, etc. When using telnet to the server, HELO reports the old server host name. This causes other servers to reject emails because it does not match the FQDM from DNS.

**Reproduction of said bug**:
1. Migrate a server to another host
2. Use the backup/restore method of migration
3. Change the host name on the target server's conf file\
4. Update and restart the target server
5. Telnet to target server, use HELO me. Server reports old host name still.

__I have tried or I do...__ 
- [x ] Tried this solution....  https://github.com/mailcow/mailcow-dockerized/issues/606
- [x ] Update script
- [x ] Restart of all containers

**System information**
I can't really provide the conf file in this case as it would expose my server host name to the public.

Further information (where applicable):

| Question | Answer |
| --- | --- |
| My operating system | Centos 7 |
| Reverse proxy (custom solution) | Nginx (using my own cert and LE is disabled in conf) |

**Comment by andryyy** [comment]
Make sure .env is a sym link to mailcow.conf

**Comment by gtmadev** [comment]
I checked and the sym link was there, but when I did a cat .env, the output was still showing what I thought was the "old" mailcow.conf. I deleted the sym link, and recreated it, but I was seeing the same thing. I checked the actual mailcow.conf and found that the old server host was there. I have no idea why or how it reverted.

I did a `docker-compose down`, removed the sym link again, and edited the mailcow.conf with the correct setting. I then recreated the sym link, and finally did a `docker-compose up -d`. This time, it all worked.

Thanks for the tip.
